infinity/code/modules/events/psi_wave.dm

```diff
@@ -0,0 +1,18 @@
+/datum/event/psi/psi_wave
+	startWhen		= 5
+	announceWhen	= 75
+
+/datum/event/psi/psi_wave/announce()
+command_announcement.Announce("� ������� ������������� ������� ���������� �������. ����������� ������� ������������� ������������ ����. ����� ��������� ������������� ������ � �������������� ������. �� ���������� ���������� �� ����� �������� ��� ����� �������� � ����������� ��������� ���-���������� ������ ���������. ��� ����������� �� ������� �������� �� �������. �� ����������� �� �������� � �������� ������ �������� ��� �������� �������� � ����. �������� ���� ��������� ������ �� ��� �������.", new_sound = GLOB.using_map.radiation_detected_sound, zlevels = affecting_z)
+
+/datum/event/psi/psi_wave/start()
+var/count = 0
+	for(var/mob/living/carbon/human/H in shuffle(GLOB.living_mob_list_))
+		if(H.client && H.stat != DEAD) && count >= rand(3, 6)
```

Comment:
if(H.client && H.stat != DEAD) && count >= rand(3, 6)
to
if((H.client && H.stat != DEAD) && count >= rand(3, 6))